src/pytest_memray/plugin.py

```diff
@@ -85,19 +87,36 @@ def __init__(self, config: Config) -> None:
         self.results: dict[str, Result] = {}
         self.config = config
         path: Path | None = config.getvalue("memray_bin_path")
+        self._tmp_dir: None | TemporaryDirectory[str] = None
         if path is None:
-            self._tmp_dir: None | TemporaryDirectory[str] = TemporaryDirectory()
-            self.result_path: Path = Path(self._tmp_dir.name)
+            # Check the MEMRAY_RESULTS_DIR environment variable. If this
```

Comment:
I don't see this environment variable getting set anywhere - what sets it?

Reply:
Oh, I see - you called it `MEMRAY_RESULTS_DIR` here but `MEMRAY_RESULT_PATH` below, throwing off my ctrl-f 😆

Reply:
Whoops, will fix it1